src/ngx_http_lua_common.h

```diff
@@ -526,6 +532,9 @@ typedef struct ngx_http_lua_ctx_s {
     unsigned         acquired_raw_req_socket:1;  /* whether a raw req socket
                                                     is acquired */
     unsigned         seen_body_data:1;
+
+    ngx_uint_t       cur_rewrite_index;
+    ngx_uint_t       cur_access_index;
```

Comment:
It's wrong to specify bigger fields after smaller fields, which may create padding holes in between.